tests/consensus/test_soft_voided.py

```diff
@@ -64,7 +66,10 @@ def test_soft_voided(self):
         self.assertEqual({txA.hash}, metaD1.voided_by)
         graphviz.labels[txD1.hash] = 'txD1'
 
-        txD2 = add_custom_tx(manager2, [(txB, 0)])
+        txD2 = gen_custom_tx(manager2, [(txB, 0)])
```

Comment:
Could you use `gen_custom_tx(manager2, [(txB, 0)], inc_timestamp=1)`? Then you don't need the other two lines below.

Reply:
Nice catch! Fixed! Thanks!